src/main/java/com/cen4010/bookstore/bookRating/BookRatingController.java

```diff
@@ -0,0 +1,20 @@
+package com.cen4010.bookstore.bookRating;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+    @RestController
+    @RequestMapping("/api/book")
```

Comment:
Can we call this something more specific than "book"? Given this is a bookRating controller, let's call it `bookRating`